sound/soc/sof/ops.h

```diff
@@ -348,6 +348,14 @@ static inline int snd_sof_dsp_send_msg(struct snd_sof_dev *sdev,
 	return sof_ops(sdev)->send_msg(sdev, msg);
 }
 
+static inline u32 snd_sof_dsp_get_ipc_version(struct snd_sof_dev *sdev)
+{
+	if (sof_ops(sdev)->get_ipc_version)
+		return sof_ops(sdev)->get_ipc_version(sdev);
+
+	return SOF_IPC_VERSION_1;
```

Comment:
The commit message refers to `SOF_INTEL_IPC_VERSION_1`
which I assume is IPC3 ?

Reply:
> The commit message refers to `SOF_INTEL_IPC_VERSION_1`
> which I assume is IPC3 ?

Yes, SOF_IPC_VERSION_1 is IPC3 and SOF_IPC_VERSION_2 is IPC4

Reply:
Would it not be confusing if we refer to the IPC3 as `SOF_IPC_VERSION_3` and IPC4 as `SOF_IPC_VERSION_4`?
But based on your other comment, the IPC4 is a a version used by closed-source firmware? Is it valid to say `SOF_IPC_VERSION_x` to a protocol which is _not_ used by SOF?